terraform/globals.tf

```diff
@@ -97,8 +91,21 @@ variable "microbosh_static_private_ip" {
   default     = "10.0.0.6"
 }
 
-/* Note: This is overridden in prod.tfvars and staging.tfvars to allow world access */
-variable "web_access_cidrs" {
-  description = "CSV of CIDR addresses for which we allow web access"
+
+/* Operators will mainly be from the office. See https://sites.google.com/a/digital.cabinet-office.gov.uk/gds-internal-it/news/aviationhouse-sourceipaddresses for details. */
+variable "admin_cidrs" {
+  description = "CSV of CIDR addresses with access to operator/admin endpoints"
   default     = "80.194.77.90/32,80.194.77.100/32,85.133.67.244/32"
 }
+
+/* Note: This is overridden in prod.tfvars to allow specific tenant access */
+variable "tenant_cidrs" {
```

Comment:
Should this be called `api_access_cidrs` to match the function of the endpoint like `web_access_cidrs`?